content/uiux/uiux.md

```diff
@@ -0,0 +1,9 @@
+---
+Title: 'UI and UX Design'
+Description: 'UI and UX Design is the practice of designing thoughtful, effective, and accessible digital experiences for web or mobile. Engineers depend heavily on the work of both UI and UX designers to help determine the flow of a particular product and how the product will look. UI and UX Design presents a great opportunity to expand into a code-related field.'
+CatalogContent:
+  - 'intro-to-ui-ux'
+  - 'paths/front-end-engineer-career-path'
+---
+
+UI and UX Design is the practice of designing thoughtful, effective, and accessible digital experiences for web or mobile. Engineers depend heavily on the work of both UI and UX designers to help determine the flow of a particular product and how the product will look. UI and UX Design presents a great opportunity to expand into a code-related field.
```

Comment:
Can we include the word "platforms" after mentioning "web and mobile"? + Last sentence, could we lowercase "design" and also rephrase it a bit. Maybe something like "Understanding UI and UX design can also open opportunities to code-related fields."

Also, when UI/UX is first mentioned can we write that out. I'm wondering what the Free Team thinks about adding this as a new topic @Dusch4593 @KTom101 @SSwiniarski @caupolicandiaz

I think it could be a good addition and UI/UX are well known terms so having them abbreviated as a topic doesn't seem too confusing for beginners.

My other thought to this concerns how the information that may pertain to UI/UX can overlap with the frontend-related topics that already exist on Docs. After taking a look at the few UIUX courses on Codecademy, I think there is a benefit to having documentation on certain concepts like wire framing and responsive design.

Ty for the team's input in advance 🙏🏻 also ty @lucwhite for thinking of this addition for discussion

Reply:
Sure thing! Yeah, this topic comes from our new [Introduction to UI and UX Design](https://www.codecademy.com/learn/intro-to-ui-ux) course. We have a bunch of entries being worked on about various design concepts (such wireframing and design methodologies).

@js6450 and I talk about this with @sonnynomnom as well.

I'll make those changes.

Reply:
Hey @yangc95 -- adding on to Lucas' comment. We met with @sonnynomnom to discuss adding UI UX as a topic to the docs. This is part of the recent Intro to UI and UX Design launch and prep for the upcoming UI UX career path. 

As Lucas said, we've got a bunch of UI UX terms coming in to add to the docs. These terms will be listed under the UI UX design category because they have design context and purposefully will be written to be tech-agnostic. The front-end related terms should remain as is.

FYI, for the term entries that CCs from our production will be submitting over the next few weeks, Lucas and I are reviewing them, so no need to check for technical accuracy.

Reply:
I think that adding a UI/UX topic makes sense especially since we just recently rolled out [a new course](https://www.codecademy.com/learn/intro-to-ui-ux) about it.

In terms of overlap, let's use [an existing Docs entry about UI design](https://www.codecademy.com/resources/docs/general/user-interface-design) (in the "General" topic) as an example. At first glance, I see two ways to go about this:

1. Preserve the General: UI Design entry and link out to any appropriate concept/term entries in the soon-to-be UI/UX topic.
2. Transplant as much info from the General: UI Design entry (into other UI/UX-topic entries) and delete afterwards (DRY things out, so to speak).

Reply:
Thanks for the clarification @lucwhite @js6450!

@Dusch4593 and I will mainly keep an eye out for formatting but will be a bit hands-off on the technical aspects then 👍

Reply:
@lucwhite @js6450 Can you make issues for the entries you all are working on so I don't accidentally work on duplicates because I am planing on making some UI entries.

Reply:
@ethan-master-coding I would hold off on working on anything related to UI/UX for the moment because @lucwhite and @js6450 are going to handle things on that front.